game/src/Game/GameSession/ProvinceOverviewPanel/ProvinceOverviewPanel.gd

```diff
@@ -122,7 +122,7 @@ func _update_info() -> void:
 
 		_rgo_name_label.text = _province_info.get(GameSingleton.get_province_info_rgo_key(),
 			GameSingleton.get_province_info_rgo_key() + _missing_suffix)
-		_rgo_icon_texture_rect.texture = GameSingleton.get_good_icon_texture(_rgo_name_label.text)
+		_rgo_icon_texture_rect.texture = null
```

Comment:
Curious why null rather than placeholder or loaded image

Reply:
That image (and most of the rest of the province overview panel) is going to be removed/redone with GUI file reading, this was the simplest temporary fix after I removed `get_good_icon_texture`.